pkg/config/user_config.go

```diff
@@ -406,7 +406,7 @@ type KeybindingCommitsConfig struct {
 	MoveDownCommit                 string `yaml:"moveDownCommit"`
 	MoveUpCommit                   string `yaml:"moveUpCommit"`
 	AmendToCommit                  string `yaml:"amendToCommit"`
-	ResetCommitAuthor              string `yaml:"resetCommitAuthor"`
+	AmendAttributeMenu             string `yaml:"amendAttributeMenu"`
```

Comment:
This is really a breaking change; if any user changed the keybinding for this command, we'll break it for them. I think it's not a big deal because I find it kind of unlikely that people remap this, but you never know.

If we really wanted to, we could automatically migrate users' config files by adding some code [here](https://github.com/jesseduffield/lazygit/blob/66b608b2f9857296645f964b67b821e431d7b807/pkg/config/app_config.go#L184), but I'm not sure it's worth it.

Reply:
I feel the same, not many people would have remapped this, but I'll add a migration to fix this just in case